src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1885,15 +1885,11 @@ static void uct_ib_mlx5_md_port_counter_set_id_init(uct_ib_mlx5_md_t *md)
 
 static int uct_ib_mlx5_check_uar(uct_ib_mlx5_md_t *md)
```

Comment:
why not call uct_ib_mlx5_devx_check_supported_uar_alloc_type() directly? uct_ib_mlx5_check_uar() looks redundant

Reply:
removed uct_ib_mlx5_check_uar, done